src/main.ts

```diff
@@ -43,7 +43,6 @@ const launchMooLite = () => {
         return;
     }
 
-    mooSocket.initialize();
```

Comment:
Maybe a bit of [Chesterton's Fence](https://en.wiktionary.org/wiki/Chesterton%27s_fence) here - is there any reason why `mooSocket.initialize` wasn't being called on construction before?